### DISCUSSION TITLE
Milvus backup & restoration not working

### DISCUSSION THREAD

**Discussion by debjotimallick** [milvus-io/milvus#29717] [Q&A and General discussion]
Hello,

We have milvus-a running in our env, we have deployed another milvus-b instance. Now we have taken backup from milvus-a and trying to restore it in milvus-b. After restoration, we found only collections are created in milvus-b with restoration process but data is not populated in any of the collections inside milvus-b. Can you please help how we should restore data from one milvus into another? We are using milvus-backup tool.

**Comment by yhmo** [comment]
Follow the steps in this discussion to backup/restore collection bwteen clusters: https://github.com/milvus-io/milvus/discussions/29630#discussioncomment-8009026
